src/js/components/Select/stories/UnSelect.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Grommet, Select, Button } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { Trash } from 'grommet-icons';
+
+const option = [
+  'One',
+  'Two',
+  'Three',
+  'Four',
+  'Five',
+  'Six',
+  'Seven',
+  'Eight',
+  'Nine',
+  'Ten',
+];
+
+const UnSelect = () => {
+  const [options] = useState(option);
+  const [value, setValue] = useState('');
+
+  const onChange = e => {
+    setValue(e.value);
+  };
+
+  const onClick = () => {
+    setValue('');
+  };
+
+  return (
+    <Grommet theme={grommet}>
+      <Box
+        fill
+        pad="medium"
+        direction="row-responsive"
+        align="center"
+        justify="center"
+      >
+        <Select
+          options={options}
+          onChange={e => onChange(e)}
+          value={value}
+          placeholder="select"
+          multiple
+        />
+        <Button onClick={onClick} icon={<Trash color="control" size="medium" />} />
```

Comment:
```suggestion
        <Button onClick={onClickClearOptions} icon={<Trash color="control" size="medium" />} />
```
I'm suggesting to rename the function, so when its defenition and implemntation appears on the top of the page, it will be implied immediatly what is its purpose.